### ISSUE TITLE
Autocomplete multiple: removeItem js function use not escaped itemlabel for selector

### ISSUE THREAD

**Issue by pacvon** [primefaces/primefaces#9175]
### Describe the bug

When try to remove an item with an apostrophe in a label from selected items, js error occures. Its because the itemValue is not escaped for use as a selector.

problem line:
var foundItem = this.multiItemContainer.children("li.ui-autocomplete-token[data-token-value='"+itemValue+"']");

this works fine"
var foundItem = this.multiItemContainer.children("li.ui-autocomplete-token[data-token-value='"+$.escapeSelector(itemValue)+"']");

### PrimeFaces version

11.0.7

### JSF implementation

MyFaces

### JSF version

2.3

### Java version

11

**Comment by melloware** [comment]
Thanks for reporting.  Fixed for 12.0.0 final.

**Comment by melloware** [comment]
MonkeyPatch:

```js
if (PrimeFaces.widget.AutoComplete) {
    PrimeFaces.widget.AutoComplete.prototype.removeItem = function(item) {
        var $this = this,
            itemValue = '';
        if ($this.input.hasClass('ui-state-disabled') || $this.input.attr("readonly")) {
            return;
        }

        if (typeof item === 'string' || item instanceof String) {
            itemValue = item;
        } else {
            itemValue = item.attr('data-token-value');
        }

        var foundItem = this.multiItemContainer.children("li.ui-autocomplete-token[data-token-value='" + $.escapeSelector(itemValue) + "']");
        if (!foundItem.length) {
            return;
        }
        var itemIndex = foundItem.index();
        if (!itemValue || itemIndex === -1) {
            return;
        }

        //remove from options
        this.hinput.children('option').eq(itemIndex).remove();

        //remove from items
        foundItem.fadeOut('fast', function() {
            var token = $(this);
            token.remove();
            $this.invokeItemUnselectBehavior(itemValue);
        });

        // if empty return placeholder
        if (this.placeholder && this.hinput.children('option').length === 0) {
            this.input.attr('placeholder', this.placeholder);
        }
    }
};
```
